helpers/spark-submit.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import pwd
+import subprocess
+
+if __name__ == "__main__":
+    os.environ["HOME"] = pwd.getpwuid(os.getuid())[5]
+
+    if os.environ.get('SPARK_HOME') is None:
+        os.environ['SPARK_HOME'] = os.environ['SNAP']
+
+    submit_args = sys.argv[1:]
+    # if '--master' not in sys.argv:
```

Comment:
I think we should set `--master` argument if user not provided it. for better UX.